web/components/templates/requestsV2/requestRow.tsx

```diff
@@ -151,6 +158,57 @@ const RequestRow = (props: {
     }
   };
 
+  const onAddScoreHandler = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setIsScoresAdding(true);
+
+    const formData = new FormData(e.currentTarget);
+    const key = formData.get("key") as string;
+    const value = formData.get("value") as any as number;
+
+    if (currentScores && currentScores[key]) {
+      setNotification("Score already exists", "error");
+      setIsScoresAdding(false);
+      return;
+    }
+
+    if (!key || !value || org?.currentOrg?.id === undefined) {
+      setNotification("Error adding score", "error");
+      setIsScoresAdding(false);
+      return;
+    }
+    try {
+      const res = await addRequestScore(
```

Comment:
Q: Can a request have more than 1 score? If im understanding right, the `addRequestScore` function adds a score to an array of scores?

Reply:
@ScottMktn Yes, request can have multiple scores but user can add scores 1 by 1 like custom properties